dtfs-central-api/api-tests/mocks/gef/gef-facilities.ts

```diff
@@ -1,47 +1,36 @@
-const { FACILITIES } = require('../../../src/constants');
+import { ObjectId } from 'mongodb';
+import { Facility, FACILITY_TYPE } from '@ukef/dtfs2-common';
+import { aFacility } from '../../../test-helpers/test-data';
 
-const baseFacilities = {
-  name: null,
-  coverStartDate: null,
-  coverEndDate: null,
-  monthsOfCover: null,
-  details: null,
-  detailsOther: null,
-  currency: null,
-  value: null,
-  coverPercentage: null,
-  interestPercentage: null,
-  paymentType: null,
-};
-
-const mockFacilities = [
+export const mockFacilities: Facility[] = [
```

Comment:
nice job with this even though not needed as part of this review

Reply:
I had already added the `aFacility` test helper in another branch but it made sense to move it here